packages/rln/src/rln.ts

```diff
@@ -140,8 +141,9 @@ export class RLNInstance {
       this._credentials = credentials;
       this._signer = signer!;
       this._contract = await RLNContract.init(this, {
-        registryAddress: registryAddress!,
-        signer: signer!
+        address: address!,
+        signer: signer!,
+        rateLimit: options.rateLimit ?? this.zerokit.getRateLimit
```

Comment:
This looks wrong 
`this.zerokit.getRateLimit` is a function so it should be called

Reply:
it's a getter :D